packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -292,12 +292,43 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
+
+    regularTick: [
+      classNames.regularTick,
+      {
+        position: 'absolute',
+        background: theme.palette.neutralTertiaryAlt,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText'
+          }
+        }
+      },
+      props.disabled && {
```

Comment:
All instances of usage like this: `props._someProp_` should be replaced by destructuring the props objetc at the beginning.